src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  width: $card-width;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 32px 16px 16px;
+  font-family: $font;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: all 0.3s;
```

Comment:
Specify the transition property, please. 
When you use all as the property value, every change to any animatable property of the element will trigger a transition. This can result in unnecessary and potentially performance-intensive transitions, especially if there are multiple properties being modified simultaneously.